crates/glaredb_core/src/functions/scalar/builtin/binary/bitand.rs

```diff
@@ -0,0 +1,146 @@
+        category: Category::Numeric,
+        description: "Performs a bitwise AND operation on two integers.",
+        arguments: &["value1", "value2"],
+        example: Some(Example {
+            example: "bitand(5, 3)",
+            output: "1",
+        }),
+    }],
+    functions: &[
+        RawScalarFunction::new(
+            &Signature::new(&[DataTypeId::Int8, DataTypeId::Int8], DataTypeId::Int8),
+            &BitAnd_::<PhysicalI8>::new(&DataType::Int8),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(&[DataTypeId::Int16, DataTypeId::Int16], DataTypeId::Int16),
+            &BitAnd_::<PhysicalI16>::new(&DataType::Int16),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(&[DataTypeId::Int32, DataTypeId::Int32], DataTypeId::Int32),
+            &BitAnd_::<PhysicalI32>::new(&DataType::Int32),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(&[DataTypeId::Int64, DataTypeId::Int64], DataTypeId::Int64),
+            &BitAnd_::<PhysicalI64>::new(&DataType::Int64),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(
+                &[DataTypeId::Int128, DataTypeId::Int128],
+                DataTypeId::Int128,
+            ),
+            &BitAnd_::<PhysicalI128>::new(&DataType::Int128),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(&[DataTypeId::UInt8, DataTypeId::UInt8], DataTypeId::UInt8),
+            &BitAnd_::<PhysicalU8>::new(&DataType::UInt8),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(
+                &[DataTypeId::UInt16, DataTypeId::UInt16],
+                DataTypeId::UInt16,
+            ),
+            &BitAnd_::<PhysicalU16>::new(&DataType::UInt16),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(
+                &[DataTypeId::UInt32, DataTypeId::UInt32],
+                DataTypeId::UInt32,
+            ),
+            &BitAnd_::<PhysicalU32>::new(&DataType::UInt32),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(
+                &[DataTypeId::UInt64, DataTypeId::UInt64],
+                DataTypeId::UInt64,
+            ),
+            &BitAnd_::<PhysicalU64>::new(&DataType::UInt64),
+        ),
+        RawScalarFunction::new(
+            &Signature::new(
+                &[DataTypeId::UInt128, DataTypeId::UInt128],
+                DataTypeId::UInt128,
+            ),
+            &BitAnd_::<PhysicalU128>::new(&DataType::UInt128),
+        ),
+    ],
+};
+
+#[derive(Debug, Clone, Copy)]
+pub struct BitAnd_<S> {
+    return_type: &'static DataType,
+    _s: PhantomData<S>,
+}
+
+impl<S> BitAnd_<S> {
+    pub const fn new(return_type: &'static DataType) -> Self {
+        BitAnd_ {
+            return_type,
+            _s: PhantomData,
+        }
+    }
+}
+
+impl<S> ScalarFunction for BitAnd_<S>
+where
+    S: MutableScalarStorage,
+    S::StorageType: BitAnd<Output = S::StorageType> + Default + Sized + Copy,
```

Comment:
```suggestion
    S::StorageType: std::ops::BitAnd<Output = S::StorageType> + Default + Sized + Copy,
```

Make this change for the rest of the functions too. Then remove the underscore.